src/app/pages/virtualization/components/instance-wizard/instance-wizard.component.html

```diff
@@ -6,22 +6,25 @@
 
 <div class="container">
   <form class="form" [formGroup]="form" (submit)="onSubmit()">
-    <ix-fieldset [title]="'Instance Configuration' | translate">
+    <ix-form-section [label]="'Instance Configuration' | translate" [help]="'Instance Configuration' | translate">
       <ix-input
+        ixRegisteredControl
         formControlName="name"
-        [label]="'Name' | translate"
+        [label]="'Name'"
         [required]="true"
       ></ix-input>
 
       <ix-checkbox
+        ixRegisteredControl
```

Comment:
The original intent was to have this directive in ix form controls and not to expose it in other places. Can't we keep doing that instead of having registeredControl here?